quickstart/src/main/java/com/twilio/video/quickstart/activity/VideoActivity.java

```diff
@@ -173,16 +172,12 @@ protected void onCreate(Bundle savedInstanceState) {
         preferences = PreferenceManager.getDefaultSharedPreferences(this);
 
         /*
-         * Enable changing the volume using the up/down keys during a conversation
+         * Setup audio management and set the volume control stream
          */
+        audioDeviceSelector = new AudioDeviceSelector(getApplicationContext());
+        savedVolumeControlStream = getVolumeControlStream();
```

Comment:
Perhaps managing the volume control stream thing we should handle in the library. Thoughts?

Reply:
Yeah I think we have a ticket for this already but yeah definitely. It might be a good addition to the `start/stop` API.